mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1087,7 +1087,7 @@ def plot_confusion_matrix():
 
     def _generate_model_predictions(self):
         """
-        Helper method for generating model predictions
+        Helper method for generating model predictions TODO: change here
```

Comment:
@sunishsheth2009 Is this the place you suggest to change the way predictions are generated?
I'm worrying about non-default evaluator's behavior. I'll try to flesh out the implementation and make sure there is no breaking change introduced.

Reply:
Yes that is what I was thinking.

Reply:
Yes